test_genie_python_using_simple.py

```diff
@@ -270,4 +270,80 @@ def test_GIVEN_waiting_for_value_above_low_limit_on_block_WHEN_block_reaches_lim
         set_pv_thread.start()
         g.waitfor_block(block=self.block_name, lowlimit=low_limit, maxwait=self.max_wait)
 
-        assert_that(set_pv_thread.is_alive(), is_(True), "Waitfor should have finished because block has changed to the limit")
\ No newline at end of file
+        assert_that(set_pv_thread.is_alive(), is_(True), "Waitfor should have finished because block has changed to the limit")
+
+class TestRunControl(unittest.TestCase):
+
+    def setUp(self):
+        g.set_instrument(None)
+        load_config_if_not_already_loaded(SIMPLE_CONFIG_NAME)
+        self.block_name = "FLOAT_BLOCK"
+        self.wait_before = 1
+        self.wait_after = 2
+        self.max_wait = (self.wait_before + self.wait_after) * 2
+        assert_that(check_block_exists(self.block_name), is_(True))
+        g.cset(self.block_name, 0)
+        g.cset(self.block_name, runcontrol=False)
+        self.block_pv = g.prefix_pv_name("CS:SB:") + self.block_name
+        g.set_pv(self.block_pv + ":AC:ENABLE", 0)
+        self._waitfor_runstate("SETUP")
+
+    def tearDown(self):
+        g.abort()
+
+    def test_GIVEN_out_of_range_block_WHEN_start_run_THEN_dae_waiting(self):
+        if g.get_runstate() != "SETUP":
```

Comment:
This check is already done at the end of setUp, is it required here?

Reply:
I think i was having some issues at some point which is why it was here, or else it was copied from somewhere else. I'll remove.

Reply:
test_genie_python_dae does it, but it didn't check in setup